app/Main.hs

```diff
@@ -84,49 +84,69 @@ nittaArgs =
     Nitta
         { filename = def &= argPos 0 &= typFile
         , output_path =
-            "gen" &= typ "PATH" &= help "Target system path"
+            "gen"
```

Comment:
hm, it's already there https://github.com/ryukzak/nitta/blob/master/fourmolu.yaml
now I don't understand why everything is changed

Reply:
Yea, it looks not very good, but I guest that it was happened because fourmolu code style isn't stable yet.  
It also looks like the configuration options aren't mature enough for us to control these changes.   https://github.com/fourmolu/fourmolu#configuration